### DISCUSSION TITLE
user support

### DISCUSSION THREAD

**Discussion by lolsenkettle** [Chaste/Chaste#297] [Q&A] [answered]
Hi, we have started using CHASTE this year. I saw that there is a hackathon and wanted to ask if we can join online to ask questions? Is the only documentation for CHASTE on github?

Many thanks, Louise Olsen

**Comment by jmosborne** [comment]
I think its fully in person. However, I'm just down the road (Melbourne Uni) and happy to have a chat to work out what you need. Note I'm more on the Cell based Side of Chaste but can probably point you in the right direction. Just email me jmosborne@unimelb.edu.au

**Comment by fcooper8472** [comment, marked as answer]
In addition to what @jmosborne has said, our main website is here:

https://chaste.github.io/

this includes:
- user guides https://chaste.github.io/docs/user-guides/
- tutorials https://chaste.github.io/docs/user-tutorials/

**Reply by lolsenkettle** [reply]
Is it possible to have access to these lecture notes: https://chaste.github.io/old_releases/release_2021.1/ChasteGuides/NmoodLectureNotes.html

**Reply by mirams** [reply]
I think they are these ones here: http://www.cs.ox.ac.uk/people/pras.pathmanathan/nmood_printable.pdf

**Reply by lolsenkettle** [reply]
Thanks these notes are very helpful
